src/pages/evm/wallet/WalletPage.vue

```diff
@@ -1,45 +1,34 @@
-<script lang="ts">
-import { defineComponent } from 'vue';
-
+<script setup lang="ts">
+import { computed, ref, watch } from 'vue';
 import { EvmToken } from 'src/antelope/types';
-
 import AppPage from 'components/evm/AppPage.vue';
 import WalletPageHeader from 'pages/evm/wallet/WalletPageHeader.vue';
 import WalletBalanceRow from 'pages/evm/wallet/WalletBalanceRow.vue';
 import { useBalancesStore, useFeedbackStore } from 'src/antelope';
 import WalletTransactionsTab from 'pages/evm/wallet/WalletTransactionsTab.vue';
 
-const feddback = useFeedbackStore();
+const feedback = useFeedbackStore();
+const tabs = ['balance', 'transactions'];
+const totalFiatAmount = ref(0);
+
+const allTokens = computed(() => useBalancesStore().loggedBalances as EvmToken[]);
+const loadingStrings = computed(() => feedback.getLoadings);
+const loading = computed(() => feedback.isLoading('updateBalancesForAccount'));
+
+watch(allTokens, (newBalances: EvmToken[]) => {
+    let newFiatBalance = 0;
+    for (let balance of newBalances){
+        newFiatBalance += parseFloat(balance.fiatBalance);
+    }
```

Comment:
Most probably `balance.fiatBalance `is coming as `undefined` and that is causing the $NaN.
Ezra and I are working on the [new Token class PR](https://github.com/telosnetwork/telos-wallet/pull/286) to remove all those inconsistencies but for now you should fix this

```suggestion
    for (let balance of newBalances){
        newFiatBalance += parseFloat(balance.fiatBalance ?? 0);
    }
```

Reply:
Pushed a fix checking that `fiatBalance` is defined before adding. This avoids unnecessarily parsing and adding 0 in all undefined cases.